ha/core/event_manager/event_manager.py

```diff
@@ -173,11 +174,11 @@ def _store_component_key(self, key: str, resource_type: str, val: list) -> None:
                        Key is: {key}')
             self._confstore.set(f'{key}', event_list)
 
-    def _store_event_key(self, resource_type: str, states: list = None, comp: str = None) -> None:
+    def _store_event_key(self, resource_type: str, states: list, comp: str = None) -> None:
```

Comment:
Why is the list = None removed from here ?

Reply:
My bad, while removing addition change I introduced in this method args, I might have removed this. I will revert it.
Thanks.

Reply:
Done.